internal/exec/stages/disks/luks.go

```diff
@@ -294,36 +300,53 @@ func (s *stage) createLuks(config types.Config) error {
 			// pass the device to clevis. We have to loop each device as
 			// the devices could be on different NICs that haven't come
 			// up yet.
+
+			// List of tang servers proven to be reachable.
+			tangServersWithoutAdv := 0
 			for _, tang := range luks.Clevis.Tang {
 				u, err := url.Parse(tang.URL)
 				if err != nil {
 					return fmt.Errorf("parsing tang URL: %v", err)
 				}
 				u.Path = path.Join(u.Path, "adv")
 				_, err = s.Fetcher.FetchToBuffer(*u, resource.FetchOptions{})
```

Comment:
Due to https://github.com/coreos/ignition/issues/1474#issuecomment-1432110813, we should only do this if no adv is given.